pkg/azurefile/controllerserver.go

```diff
@@ -278,6 +303,10 @@ func (d *Driver) DeleteSnapshot(ctx context.Context, req *csi.DeleteSnapshotRequ
 
 	_, err = shareURL.WithSnapshot(snapshot).Delete(ctx, azfile.DeleteSnapshotsOptionNone)
 	if err != nil {
+		if strings.Contains(err.Error(), "ShareSnapshotNotFound") {
+			klog.V(2).Infof("the specify snapshot(%s) was not found", snapshot)
```

Comment:
klog.V(2).Warningf("specified snapshot(%s) was not found", snapshot)

Reply:
done.